lib/views/file-patch-header-view.js

```diff
@@ -25,9 +26,15 @@ export default class FilePatchHeaderView extends React.Component {
     undoLastDiscard: PropTypes.func.isRequired,
     diveIntoMirrorPatch: PropTypes.func.isRequired,
     openFile: PropTypes.func.isRequired,
+    // should probably change 'toggleFile' to 'toggleFileStagingStatus'
+    // because the addition of another toggling function makes the old name confusing.
```

Comment:
Honestly I've never been happy with the "toggle" function names. None of the alternatives I've been able to come up with are succinct enough though. We need a nice, single verb that means "stage or unstage"...